deployment-server/Setup.hs

```diff
@@ -0,0 +1,2 @@
+import           Distribution.Simple
```

Comment:
We don't need this file, do we?

Reply:
I never change Setup.hs, is there any reason to?

Reply:
I think you can just delete it.

Reply:
Hm, we have a bunch of pointless `Setup.hs` lying around our repo, in fact.

Reply:
Ok, I'll take this as an opportunity to remove unnecessary ones

Reply:
done